var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -77,7 +77,7 @@ class Adios2(CMakePackage):
     # depends_on('googletest')
     # depends_on('pugixml')
     # depends_on('kwsys')
-    # depends_on('nlohmannjson')
+    # depends_on('nlohmann-json')
```

Comment:
just to document the proper package name for later external package support.

prepared in https://github.com/ornladios/ADIOS2/pull/539